test/index.test.ts

```diff
@@ -276,7 +284,7 @@ describe('Probot', () => {
     })
 
     it('sets throttleOptions', async () => {
-      const probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken' })
+      probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken' })
 
       expect(probot.throttleOptions.Bottleneck).toBe(Bottleneck)
       expect(probot.throttleOptions.connection).toBeInstanceOf(Bottleneck.IORedisConnection)
```

Comment:
Property 'throttleOptions' is private and only accessible within class 'Probot'.
Property 'IORedisConnection' does not exist on type 'typeof import("bottleneck")'.